rocAL/rocAL/include/augmentations/color_augmentations/node_blur.h

```diff
@@ -26,19 +26,16 @@ THE SOFTWARE.
 #include "parameter_vx.h"
 #include "graph.h"
 
-class BlurNode : public Node
-{
+class BlurNode : public Node {
 public:
     BlurNode(const std::vector<Tensor *> &inputs, const std::vector<Tensor *> &outputs);
     BlurNode() = delete;
-    void init(int sdev);
-    void init(IntParam *sdev);
-
+    void init(int kernel_size);
+    void init(IntParam *kernel_size);
```

Comment:
Please check if it is better to add suffix/prefix to kernel_size for IntParam to differentiate from fixed case
like kernel_size_param